lib/ansible/cli/doc.py

```diff
@@ -423,7 +423,7 @@ def _format(string, *args):
         if style not in ref_style and style.upper() not in STYLE and style not in C.COLOR_CODES:
             raise KeyError("Invalid format value supplied: %s" % style)
 
-        if C.ANSIBLE_NOCOLOR:
+        if C.ANSIBLE_NOCOLOR or not ANSIBLE_COLOR:
```

Comment:
```suggestion
        if not ANSIBLE_COLOR:
```
`ANSIBLE_COLOR` already respects the config, there's no reason to check it again here.